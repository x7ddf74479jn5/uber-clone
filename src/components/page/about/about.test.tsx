import { render } from "@testing-library/react";

import { About } from ".";

describe("About page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<About />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
