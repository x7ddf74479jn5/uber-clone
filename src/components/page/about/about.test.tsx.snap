// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<DocumentFragment>
  <h2>
    About
  </h2>
</DocumentFragment>
`;
